common/daemon.c

```diff
@@ -119,6 +121,21 @@ static void add_steal_notifiers(const tal_t *root)
 }
 #endif
 
+static void *wally_tal(size_t size)
+{
+	return tal_arr_label(wally_tal_ctx, u8, size, "wally");
```

Comment:
Pretty sure that we changed this later to just call it "wally_notleak"?

Reply:
ah yes, updated!